app/views/interstitial/DisplayAddressInterstitialView.scala.html

```diff
@@ -33,6 +33,6 @@
 
     @h1("label.profile")
     @p(Text(messages("label.you_can_see_this_part_of_your_account_if_you_complete")))
-    @button(msg = "global.label.continue_button", href=Some(controllers.routes.ApplicationController.uplift(Some(SafeRedirectUrl(continueUrl))).url))
+    @button(msg = "global.label.continue_button", href=Some(controllers.routes.ApplicationController.uplift(Some(RedirectUrl(continueUrl))).url))
```

Comment:
The import needs to be updated to @import uk.gov.hmrc.play.bootstrap.binders.RedirectUrl